### ISSUE TITLE
Modal dialog min-width issue: cutoff in thinner browser windows

### ISSUE THREAD

**Issue by tslaton** [dunovank/jupyter-themes#224]
My experience so far with this extension has mostly been pretty great! 

However, one issue in particular has been bugging me. Whenever any modal shows up - be it the rename modal, the deletion modal, or something else - it is cutoff when my browser windows are approximately half-screen width. It looks fine in full-screen mode. I don't have this issue with the default theme, where it looks like the modal width is percentage-based. I assume that's the fix. Looking at the page in Chrome's dev-tools, I see that the `modal-dialog` class appears to govern these modals. Its `min-width` is set to 950px. Removing that property or setting it to a percentage width appears to fix the problem.

Here is an image illustrating the issue, comparing the default theme with my dark theme setup at half-screen width:

<img width="838" alt="screen shot 2018-02-01 at 10 31 46 pm" src="https://user-images.githubusercontent.com/9665562/35719914-55c44668-07a0-11e8-9ac1-d8674c1e0cf5.png">

My settings for `jupyter-themes` are as described in the screenshot, namely `jt -t onedork -altout -cellw 88% -vim`. No combination of existing options I'm aware of affects modal width at all.

**Comment by tlkh** [comment]
Agreed, I don't think there's a config for this. We probably need to edit the CSS!
commenting out `min-width` in custom.css fixes this.
`
modal .modal-dialog {
    /* min-width: 950px; */
`
